### DISCUSSION TITLE
v18.22.9-5.3.11 - CercubePlusExtra crashing immediately on launch

### DISCUSSION THREAD

**Discussion by BlueJeans7** [arichornlover/YouTubeRebornPlus#55] [General]
Is it just me, or is anyone else experiencing a crashing issue? I initially updated the app from the previous version and then completely uninstalled it before reinstalling this version. However, the app continues to crash upon launch.

iOS version: 16.5
Device: iPhone 13
Sideloaded method: SideStore

Please let me know if you require any additional details from my end.

**Comment by BlueJeans7** [comment]
Today, I installed from this repository and everything worked seamlessly! I'm still unsure what the issue was yesterday. Additionally, I'm curious to know if installing from this repository will also include extensions by default.

https://raw.githubusercontent.com/CercubeSource/CercubePlusAltstore/main/apps.json

I can no longer find the option to hide this button.
![IMG_3980](https://github.com/arichorn/YouTubeRebornPlus/assets/24786247/9aaa69d5-4ec9-4d20-a1c1-1b4be956d211)

**Reply by arichornlover** [reply]
That’s strange? The hide Cercube button option is in Settings > CercubePlusExtra > Miscellaneous

**Reply by arichornlover** [reply]
Next release of CercubePlusExtra, the issue you have will be gone. I found a patch that would fix the buttons. So you would be able to hide them correctly and also it will no longer look weird in the screenshot.

**Comment by arichornlover** [comment]
> Is it just me, or is anyone else experiencing a crashing issue? I initially updated the app from the previous version and then completely uninstalled it before reinstalling this version. However, the app continues to crash upon launch.
iOS version: 16.5 Device: iPhone 13 Sideloaded method: SideStore
Please let me know if you require any additional details from my end.

Wait so is it still crashing only on that version? If it is still crashing then I will remove the newly added tweak YTHoldForSpeed.

> Today, I installed from this repository and everything worked seamlessly! I'm still unsure what the issue was yesterday. Additionally, I'm curious to know if installing from this repository will also include extensions by default.

I honestly don’t know. Since I’m not an Altstore 2.0 Beta Tester then i cannot test it or confirm it.
